docs/webhooks/index.html

```diff
@@ -251,11 +269,75 @@ <h3 id="payloads">Webhook payloads</h3>
             <div class="section">
                 <h2>Events for CodeSignal Test</h2>
  <div class="subsection">
-<h3 id="testworkflow">Test webhooks process flow</h3>
-<img alt="Diagram of webhooks process flow for CodeSignal Test" width="100%" src="https://codesignal.com/wp-content/uploads/2021/03/webhooksproces_test.png" />
+     <h3 id="candidateworkflow">Candidate workflow</h3><br>
+<img alt="Diagram of webhooks triggered on the candidate flow for CodeSignal Test" width="100%" src="../images/candidate-events.png" />
 </div>
                 <div class="subsection">
-                    <h3 id="companyTestSessionStarted">companyTestSessionStarted</h3>
+                    <h4 id="preScreenCandidateDeclined">preScreenCandidateDeclined</h4>
+                    <p>
+                        Fired when a candidate declines a test session invitation.
+                    </p>
+                    <div>Request format:</div>
+                    <pre><code>{
+  event: 'preScreenCandidateDeclined',
+  triggeredOn: number,
+  payload: {
+    testSessionId: string,
+    testId: string,
+    testTitle: string,
+    candidateEmail: string,
+    candidateName: string,
+    externalId?: string
+  }
+};</code></pre>
+                    <div>Sample request:</div>
+                    <pre><code>{
+  event: 'preScreenCandidateDeclined',
+  triggeredOn: 1553720789347,
+  payload: {
+    testSessionId: 'lehu382hdleh29',
+    testId: 'dhey29hcl28hl28',
+    testTitle: 'Software Engineer intern',
+    candidateEmail: 'john.doe@gmail.com',
+    candidateName: 'John Doe'
+  }
+};</code></pre>
+                </div>
+                <div class="subsection">
+                    <h4 id="preScreenExpired">preScreenExpired</h4>
+                    <p>
+                        Fired when a test session expires.
+                    </p>
+                    <div>Request format:</div>
+                    <pre><code>{
+  event: 'preScreenExpired',
+  triggeredOn: number,
+  payload: {
+    testSessionId: string,
+    testId: string,
+    testTitle: string,
+    candidateEmail: string,
+    candidateName: string,
+    expiredReason: string,
+    externalId?: string
+  }
+};</code></pre>
+                    <div>Sample request:</div>
+                    <pre><code>{
+  event: 'preScreenExpired',
+  triggeredOn: 1553720789347,
+  payload: {
+    testSessionId: 'lehu382hdleh29',
+    testId: 'dhey29hcl28hl28',
+    testTitle: 'Software Engineer intern',
+    candidateEmail: 'john.doe@gmail.com',
+    candidateName: 'John Doe',
+    expiredReason: 'testNotTaken'
```

Comment:
[related to my comment above] This feels like either it should be human-readable _or_ we should provide an enumerated list of possible "terse" versions. Right now it's a slightly cryptic code, which will be a bit hard for consumers to interpret without context.